services/jira/jira-test-helpers.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const sinon = require('sinon')
+const serverSecrets = require('../../lib/server-secrets')
+
+const user = 'admin'
+const pass = 'password'
+
+function mockJiraCreds() {
+  serverSecrets['jira_username'] = undefined
+  serverSecrets['jira_password'] = undefined
+  sinon.stub(serverSecrets, 'jira_username').value(user)
+  sinon.stub(serverSecrets, 'jira_password').value(pass)
+}
+
+function restore() {
+  sinon.restore()
```

Comment:
It looks like this function is not intended to be used publicly: https://github.com/sinonjs/sinon/issues/1390

Reply:
Since sinon@5.x the default object is itself a sandbox, and most of the documentation says to use that directly for the majority of use cases, https://sinonjs.org/releases/v7.2.2/sandbox/. 

They show this pattern, including the `restore` function on the docs for migrating to sinon@5.x https://sinonjs.org/guides/migrating-to-5.0.html. 

In that issue I see references to sinon@2.x/4.x so maybe that's changed?